cloud/docker.go

```diff
@@ -142,12 +142,28 @@ func (m *dockerManager) GetInstanceStatus(ctx context.Context, h *host.Host) (Cl
 		if client.IsErrConnectionFailed(err) {
 			return StatusTerminated, nil
 		}
-		return StatusUnknown, errors.Wrapf(err, "Failed to get container information for host '%v'", h.Id)
+		return StatusUnknown, errors.Wrapf(err, "Failed to get container information for host '%s'", h.Id)
 	}
-
 	return toEvgStatus(container.State), nil
 }
 
+func (m *dockerManager) SetPortMappings(ctx context.Context, h *host.Host, parent *host.Host) error {
+	container, err := m.client.GetContainer(ctx, parent, h.Id)
```

Comment:
Do we ever use anything other than `h.Id` from `h`? If not, we can take advantage of the compiler and only pass a string, eliminating the risk of swapping the two `*host.Host` arguments.

Reply:
Oh, that's right. I missed that. I think this is awkward, but I can't of something that would make it better. Since there are two host documents, we might just be stuck with two host documents in the signature.

However! Remember that you can write `h, parent *host.Host`.